src/xdocs/config_whitespace.xml

```diff
@@ -1280,7 +1280,7 @@ public void foo(final char @NotNull [] param) {} // No violation
                   GENERIC_END</a>,
                 <a href="apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#ELLIPSIS">
                   ELLIPSIS</a>,
-                <a href="apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#ELLIPSIS">
+                <a href="apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#METHOD_REF">
                   METHOD_REF</a>.
```

Comment:
Issue must be because our UT doesn't validate the URL. It would be great if someone could expand the UT to also check the URL then we wouldn't have this problem again.

Reply:
I am willing to try it, after I finish my proposal of GSoC Checkstyle recently.